README.md

```diff
@@ -1,2 +1,37 @@
 # ros_lpms_imu
-A ROS driver for LP-RESEARCH IMU sensors.
+
+A ROS driver for [LP-RESEARCH IMU sensors](https://www.lp-research.com/products/).
+
+## Installation
+
+Currently, the driver needs to be installed from source. Clone the [timesync package](https://github.com/larics/timesync_ros) (a utility for providing more accurate timestamps for ROS messages) and the driver into the `src` subfolder of your [catkin workspace](http://wiki.ros.org/catkin/Tutorials/create_a_workspace) and run `catkin_make`:
+
+```
+roscd
+cd ../src
+git clone https://github.com/larics/timesync_ros.git
+git clone https://github.com/larics/lpms_imu.git
+cd ..
+catkin_make
+```
+
+## Using the driver
+
+Assuming you have `roscore` running, just rosrun the driver:
+
+```
+rosrun lpms_imu lpms_imu_node
+```
+
+The driver assumes your sensor is connected to the `/dev/ttyUSB0` port. If this is not the case, use the `_port` parameter, e.g.
```

Comment:
It also makes sense to documenta the _sensor_model parameter. I.e.

The driver assumes that you are using a LPMS-U2 unit connected to `/dev/ttyUSB0`.  You can change this using the `_sensor_model` and `_port` parameters.  `_sensor_model` takes the values `DEVICE_LPMS_B`, `DEVICE_LPMS_U`, `DEVICE_LPMS_C`, `DEVICE_LPMS_BLE`, `DEVICE_LPMS_RS232`, `DEVICE_LPMS_B2`, `DEVICE_LPMS_U2`, and `DEVICE_LPMS_C2`. The possible values for `_port` then depend on the type of IMU.  For instance, a Bluetooth device could be used by issuing

```
rosrun lpms_imu lpms_imu_node _sensor_mode:=DEVICE_LPMS_B2 _port:=00:11:22:33:44:55
```